t/34-developer_mode-unit.t

```diff
@@ -650,13 +650,6 @@ $t_livehandler->ua($ua);
 $t_livehandler->app($app);
 
 subtest 'websocket proxy (connection from client to live view handler not mocked)' => sub {
-    # dumps the state of the websocket connections established in the following subtests
-    # note: not actually used after all, but useful during development
-    sub dump_websocket_state {
-        print('finished: ' . Dumper($t_livehandler->{finished}) . "\n");
-        print('messages: ' . Dumper($t_livehandler->{messages}) . "\n");
```

Comment:
This looks to me like something I'd move out of the subtest and guard behind a debug flag if it's useful enough. Right here you would be very limited to what you can do with it.

Reply:
Are you just thinking out loud or expecting me to change anything?

Reply:
I meant to address this to @Martchus since he introduced the code and might want to re-introduce something more flexible as a follow-up